ui/components/Header.js

```diff
@@ -288,7 +288,7 @@ function K8sContextMenu({
     position: 'absolute',
     left: '-7rem',
     zIndex: '-1',
-    bottom: showFullContextMenu ? '-55%' : '-110%',
+    bottom: showFullContextMenu ? '40%' : '-110%',
```

Comment:
This should be a factor of the size of the nav header.

Reply:
Let's merge this, but hold out hope that you will follow up with either:
1. relative positioning (ideal).
2. a calc to ensure that if/when the header changes size, that this adjusts (not ideal).